pkg/events/events.go

```diff
@@ -6401,5 +6402,28 @@ var Definitions = eventDefinitions{
 				{Type: "[]trace.DnsResponseData", Name: "dns_response"},
 			},
 		},
+		PrintMemDump: {
+			ID32Bit: sys32undefined,
+			Name:    "print_mem_dump",
+			Sets:    []string{},
+			Params: []trace.ArgMeta{
+				{Type: "void*", Name: "address"},
+				{Type: "bytes", Name: "bytes"},
+				{Type: "char*", Name: "arch"},
+				{Type: "char*", Name: "symbol_name"},
+				{Type: "char*", Name: "symbol_owner"},
+			},
+			Probes: []probeDependency{
+				{Handle: probes.PrintMemDump, Required: true},
+			},
+			Dependencies: dependencies{
+				Events: []eventDependency{
+					{EventID: DoInitModule},
+				},
+				KSymbols: []kSymbolDependency{
+					{Symbol: "sys_call_table", Required: true},
```

Comment:
Is this needed? I don't see the symbol used anywhere else in the PR.

Reply:
We need the kernel symbols to be able to translate the provided symbol to address.
This is how I managed to tell tracee-ebpf that there is a dependancy.
Now that I think of it there will be a problem with the lazy load...
@NDStrahilevitz what do you recommend?

Reply:
Oh I see, this forces tracee to construct the kernelSymbols object.
Lazy version shouldn't work different actually, but we need some way to declare a dependency for the event to require symbol access in general.
Maybe `KSymbols` should be a struct of:
```
     RequireDynamicSymbols bool
     StaticSymbols                   []kSymbolDependency
```

Then for events that only need dynamic access we mark the bool. If we need some preloaded static symbol, we add it to the nested array.

Reply:
@yanivagman What are your thoughts? I'm fine with it if you say it's ok - I'll implement

Reply:
It seems good to me too

Reply:
Dependencies section of an Event should well define the needs of the event to work correctly. If it is missing a way to define that symbols are required in general, then I think the approach @NDStrahilevitz suggested is good